solana-common/src/block.rs

```diff
@@ -0,0 +1,31 @@
+use substreams_solana::pb::sf::solana::r#type::v1::Block;
+
+// Vote111111111111111111111111111111111111111
+static VOTE_INSTRUCTION: &'static [u8] = &[7, 97, 72, 29, 53, 116, 116, 187, 124, 77, 118, 36, 235, 211, 189, 179, 216, 53, 94, 115, 209, 16, 67, 252, 13, 163, 83, 128, 0, 0, 0, 0];
+
```

Comment:
We have `bs58!` now:

```suggestion
static VOTE_INSTRUCTION: [u8; 32] = b58!("Vote111111111111111111111111111111111111111");
```

You will need to change the `contains` condition below too to be `message.account_keys.iter().any(|v| v == &VOTE_INSTRUCTION)`